synapse/rest/client/v1/pusher.py

```diff
@@ -107,6 +108,21 @@ async def on_POST(self, request):
         logger.debug("set pushkey %s to kind %s", content["pushkey"], content["kind"])
         logger.debug("Got pushers request with body: %r", content)
 
+        # rewrite push url from m.org sygnal to Tchap sygnal
```

Comment:
We already discussed about a change about this code, but I failed to find where, so I post here a comment:
We agreed to use the same url "https://sygnal.tchap.gouv.fr" on both platforms (Preprod and Prod), because this url will be internal. This should simplify the patch